app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, old_file, path = command.split()
```

Comment:
You also need to check that the command is `mv` and two afterward arguments are not the same.